.mailmap

```diff
@@ -1,26 +1,38 @@
+Alexander Hulpke <hulpke@math.colostate.edu> <hulpke@colostate.edu>
+Alexander Konovalov <alexander.konovalov@st-andrews.ac.uk> <alex-konovalov@users.noreply.github.com>
 Alexander Konovalov <alexander.konovalov@st-andrews.ac.uk> <alexk@cs.st-andrews.ac.uk>
 Alexander Konovalov <alexander.konovalov@st-andrews.ac.uk> <alexk@mcs.st-andrews.ac.uk>
-Alexander Konovalov <alexander.konovalov@st-andrews.ac.uk> <alex-konovalov@users.noreply.github.com>
 Andreas Distler <adistler@fc.ul.pt> <andreas@mcs.st-and.ac.uk>
 Burkhard Höfling <burkhard@hoefling.name> <bh11@users.noreply.github.com>
 Chris Jefferson <caj21@st-andrews.ac.uk>
 Chris Jefferson <caj21@st-andrews.ac.uk> <chris@bubblescope.net>
 Chris Wensley <cdwensley.maths@btinternet.com> <cwensley1234@btinternet.com>
 Christopher Russell <cr66@st-andrews.ac.uk>
+Dmitrii Pasechnik <dimpase@gmail.com> <dima@pasechnik.info>
 Dmitrii Pasechnik <dimpase@gmail.com> <Dmitrii.Pasechnik@cs.ox.ac.uk>
```

Comment:
@dimpase please let me know if `Dmitrii Pasechnik <dimpase@gmail.com>` is OK as your "preferred" git "identity" (or feel free to suggest something else)

Reply:
Please use `Dmitrii Pasechnik <dima@pasechnik.info>`. Thanks for taking care of this.